tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java

```diff
@@ -282,6 +283,28 @@ public TiIndexInfo getPrimaryKey() {
     return null;
   }
 
+  public List<TiIndexColumn> convertIndexColToPrefixCols(TiIndexInfo indexInfo) {
```

Comment:
Shall we add some tests for prefix length cluster index? just like ` a varchar(50), primary key (a (20))` and insert a value with 50 characters.

Reply:
Add more test